public/app/plugins/datasource/prometheus/datasource.ts

```diff
@@ -56,6 +58,7 @@ export class PrometheusDatasource extends DataSourceApi<PromQuery, PromOptions>
   queryTimeout: string;
   httpMethod: string;
   languageProvider: PrometheusLanguageProvider;
+  exemplarTraceIDDestination: ExemplarTraceIDDestination | undefined;
```

Comment:
So right now we can only add one datalink for the exemplars right?

Reply:
Changed it to support multiple link configuration